openfisca_tasks/install.mk

```diff
@@ -1,13 +1,19 @@
+## Install overall dependencies
+install-deps:
+	@$(call print_help,$@:)
+	@python -m pip install --upgrade pip twine wheel
+	@python -m pip install --upgrade --no-dependencies openfisca-country-template
+	@python -m pip install --upgrade --no-dependencies openfisca-extension-template
+
 ## Install project dependencies.
-install:
+install: install-deps
```

Comment:
If the template packages are only needed for testing, should we really install them systematically?

Reply:
Well, I assume people installing from source using make are in that use case.